core/src/main/java/org/mskcc/cbio/portal/util/GlobalProperties.java

```diff
@@ -658,6 +660,21 @@ public static String getDigitalSlideArchiveMetaUrl(String caseId)
         return new String[] {url.replace("{cancer.type}", typeOfCancer)};
     }
 
+    public static boolean useInternalPathReports()
+    {
+        boolean useInternalPathologyReport = false;
+        String propertyFlag = properties.getProperty(PATIENT_VIEW_USE_INTERNAL_PATH_REPORT);
+        if (propertyFlag != null) {
```

Comment:
You can simplify this quite a bit because `getProperty` can have a [default value](https://docs.oracle.com/javase/7/docs/api/java/util/Properties.html#getProperty(java.lang.String,%20java.lang.String)).